tutorials/intro/40_sensor_locations.py

```diff
@@ -178,56 +168,56 @@
 ax3d.view_init(azim=70, elev=15)
 
 ###############################################################################
-# It's probably evident from the 2D topomap above that there is some
-# irregularity in the EEG sensor positions in the :ref:`sample dataset
-# <sample-dataset>` — this is because the sensor positions in that dataset are
-# digitizations of the sensor positions on an actual subject's head, rather
-# than idealized sensor positions based on a spherical head model. Depending on
-# what system was used to digitize the electrode positions (e.g., a Polhemus
-# Fastrak digitizer), you must use different montage reading functions (see
-# :ref:`dig-formats`). The resulting :class:`montage <mne.channels.DigMontage>`
-# can then be added to :class:`~mne.io.Raw` objects by passing it to the
-# :meth:`~mne.io.Raw.set_montage` method (just as we did above with the name of
-# the idealized montage ``'standard_1020'``). Once loaded, locations can be
-# plotted with :meth:`~mne.channels.DigMontage.plot` and saved with
-# :meth:`~mne.channels.DigMontage.save`, like when working with a standard
-# montage.
+# The previous 2D topomap reveals irregularities in the EEG sensor positions in
+# the :ref:`sample dataset <sample-dataset>` — this is because the sensor
+# positions in that dataset are digitizations of actual sensor positions on the
+# head rather than idealized sensor positions based on a spherical head model.
+# Depending on the digitization device (e.g., a Polhemus Fastrak digitizer),
+# you need to use different montage reading functions (see :ref:`dig-formats`).
+# The resulting :class:`montage <mne.channels.DigMontage>` can then be added to
+# :class:`~mne.io.Raw` objects by passing it as an argument to the
+# :meth:`~mne.io.Raw.set_montage` method (just as we did before with the name
+# of the predefined ``'standard_1020'`` montage). Once loaded, locations can be
+# plotted with the :meth:`~mne.channels.DigMontage.plot` and saved with the
+# :meth:`~mne.channels.DigMontage.save` methods of the
+# :class:`montage <mne.channels.DigMontage>` object.
 #
 # .. note::
 #
-#     When setting a montage with :meth:`~mne.io.Raw.set_montage`
-#     the measurement info is updated in two places (the ``chs``
-#     and ``dig`` entries are updated). See :ref:`tut-info-class`.
-#     ``dig`` may contain HPI, fiducial, or head shape points in
-#     addition to electrode locations.
+#     When setting a montage with :meth:`~mne.io.Raw.set_montage`,
+#     the measurement info is updated in two places (both ``chs``
+#     and ``dig`` entries are updated) – see :ref:`tut-info-class` for more
+#     details. Note that ``dig`` may contain HPI, fiducial, or head shape
+#     points in addition to electrode locations.
 #
 #
-# Rendering sensor position with mayavi
+# Rendering sensor position with Mayavi
```

Comment:
it's not mayavi anymore actually. I would say with 3D OpenGL maybe?

Reply:
Is it VTK then? I think we should name the package we're using, because technically it might not be rendered with OpenGL (Vulcan, Metal, ...).

Reply:
since the dependency may change in time, perhaps we should just say "Visualizing sensors in 3D surface renderings" or something.